src/site/site.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project name="${project.name}" 
-  xmlns="http://maven.apache.org/DECORATION/1.4.0" 
+    xmlns="http://maven.apache.org/DECORATION/1.7.0" 
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/DECORATION/1.4.0 http://maven.apache.org/xsd/decoration-1.4.0.xsd">
+  xsi:schemaLocation="http://maven.apache.org/DECORATION/1.7.0 http://maven.apache.org/xsd/decoration-1.7.0.xsd">
 
   <skin>
     <groupId>org.apache.maven.skins</groupId>
     <artifactId>maven-fluido-skin</artifactId>
-    <version>1.3.1</version>
+    <version>1.5</version>
```

Comment:
@bpross-52n Is there a reason that you introduced version 1.5? The newest available version is 2.0.1 and it might be better to go to this version (or at least to the latest version of the 1.x line: 1.12.0).